demo/src/app/components/demo/badges/badges.ts

```diff
@@ -6,11 +6,11 @@ import {Component} from '@angular/core';
 })
 export class DemoBadges {
 
-  type = 'inverse';
+  type = 'default';
 
-  private types: string[] = ['default', 'shade', 'inverse'];
+  private types: string[] = ['default', 'shade', 'inverse', 'alt-inverse', 'success', 'info', 'warning', 'error', 'offline', 'alert-texture'];
 
   change() {
-    this.type = this.types[(this.types.indexOf(this.type) + 1) % 3];
+    this.type = this.types[(this.types.indexOf(this.type) + 1) % 10];
```

Comment:
Can you use `this.types.length` instead of the actual number, now that they are increasing? :)